py3-networkx.yaml

```diff
@@ -11,10 +11,10 @@ package:
 vars:
   pypi-package: networkx
 
+# Removed 3.10 because "Package 'networkx' requires a different Python: 3.10.17 not in '>=3.11'"
 data:
   - name: py-versions
     items:
-      3.10: '310'
```

Comment:
Do we know if this will break any other packages that pull this in as a dependency?